MediaManager/Platforms/Apple/Notifications/NotificationManager.cs

```diff
@@ -16,8 +16,8 @@ public override bool Enabled
             set
             {
                 base.Enabled = value;
-                ShowPlayPauseControls = false;
-                ShowNavigationControls = false;
+                ShowPlayPauseControls = true;
```

Comment:
Aah i see now. This should use `value` as set on both.

Reply:
I think you are right